sdk/rust/src/sqlite.rs

```diff
@@ -0,0 +1,112 @@
+wit_bindgen_rust::import!("../../wit/ephemeral/sqlite.wit");
+
+use sqlite::Connection as RawConnection;
+
+/// Errors which may be raised by the methods of `Store`
+pub type Error = sqlite::Error;
+
+/// A parameter used when executing a sqlite statement
+pub type DataTypeParam<'a> = sqlite::ValueParam<'a>;
+/// A single column's result from a database query
+pub type DataTypeResult = sqlite::ValueResult;
+
+/// Represents a store in which key value tuples may be placed
+#[derive(Debug)]
+pub struct Connection(RawConnection);
+
+impl Connection {
+    /// Open a connection
+    pub fn open(database: &str) -> Result<Self, Error> {
```

Comment:
Could we add an `open_default()` please, to save folks having to use the magic string everywhere, and for consistency with the Rust KV SDK?  Thanks!